pkg/system/phase4_configuring.go

```diff
@@ -682,6 +685,49 @@ func (r *Reconciler) RegisterToCluster() error {
 	return r.NBClient.RegisterToCluster()
 }
 
+// ReconcileDefaultNamespaceStore checks if the noobaa is running on Fusion HCI with
+// spectrum scale and attempts to the default nsfsnamespacestore using spectrum scale
+// storage class.
+func (r *Reconciler) ReconcileDefaultNamespaceStore() error {
+	// Skip if joining another NooBaa
+	if r.JoinSecret != nil {
+		return nil
+	}
+
+	// Check if ODF is installed on Fusion HCI cluster with spectrum scale.
+	if !util.IsFusionHCIWithScale() {
+		r.Logger.Info(" IBM Fusion HCI with spectrum scale NOT detected.")
+		return nil
+	}
+
+	log := r.Logger.WithField("func", "ReconcileDefaultNamespaceStore")
+
+	util.KubeCheck(r.DefaultNamespaceStore)
+	// Namespacestore already exist
+
+	if r.DefaultNamespaceStore.UID != "" {
```

Comment:
I think if we have DefaultNamespaceStore then that means we have DefaultNsfsPvc here.
What do you think @jackyalbo ?

Reply:
Can you have a small test checking what happens if the pvc is already created. if it failes the create then we should check, as it will stop us from creating the backingstore. if not, leave it like this.
another thing I can see missing also is this check: `if r.NooBaa.Spec.ManualDefaultBackingStore`
if user ask us not to create defaultBackingStore so we should also skip the defaultNamespaceStore.

Reply:
I think we can remove following check from Namespacestore reconciler.
if r.NooBaa.Spec.ManualDefaultBackingStore

If it is true default NSFS PVC will not be created and that mean we will not reconcile Namespacestore.